src/MLJCatBoostInterface.jl

```diff
@@ -72,12 +72,30 @@ end
 include("mlj_catboostclassifier.jl")
 include("mlj_catboostregressor.jl")
 
-const CATBOOST_MODELS = Union{CatBoostClassifier,CatBoostRegressor}
+const CatBoostModels = Union{CatBoostClassifier,CatBoostRegressor}
+
+function MMI.selectrows(::CatBoostModels, I, X, y)
```

Comment:
The "data " part of this signature should match the new signature of `fit`. So you are missing the two extra items `cat_features` and `text_features`, which you will return untouched as the last two return values.

Reply:
Done!